acct-group/motion/metadata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="project">
+        <email>media-video@gentoo.org</email>
+    </maintainer>
```

Comment:
You can't add them here without their consent. Did you ask? If not, put yourself and proxy-maint project here. 
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml

Reply:
No, have not asked  them; I assumed because they are the maintainer of media-video/motion that they'd also maintain the uid/gid. I will change the maintainer to me and the proxy maint group.